src/platform/user/profile/vet360/actions/transactions.js

```diff
@@ -198,10 +198,11 @@ export const validateAddress = (
       : await localVet360.addressValidationSuccess();
     const { addresses } = response;
     const suggestedAddresses = addresses
-      .filter(
-        address =>
-          address.addressMetaData.deliveryPointValidation === 'CONFIRMED' &&
-          address.addressMetaData.confidenceScore >= 80,
+      // sort highest confidence score to lowest confidence score
+      .sort(
+        (firstAddress, secondAddress) =>
+          secondAddress.addressMetaData.confidenceScore -
```

Comment:
Should we use optional chaining here in case `secondAddress` and `firstAddress` are missing kv pairs? :)